src/toil/batchSystems/kubernetes.py

```diff
@@ -492,7 +512,7 @@ def _isPodStuckOOM(self, podObject, minFreeBytes=1024 * 1024 * 2):
 
         # Look for it
         # TODO: When the Kubernetes Python API actually wraps the metrics API, switch to that
-        response = self._api('customObjects').list_namespaced_custom_object('metrics.k8s.io', 'v1beta1',
+        response = self.try_kubernetes(self._api('customObjects').list_namespaced_custom_object, 'metrics.k8s.io', 'v1beta1',
```

Comment:
This line is a bit long and has incurred the horizontal-scrolling ire of Github's PR display. Some of the args on the first line should move down to their own line(s).